graylog2-web-interface/src/components/nodes/NodeMaintenanceDropdown.jsx

```diff
@@ -29,6 +29,10 @@ const NodeMaintenanceDropdown = React.createClass({
               <MenuItem>Configure internal logging</MenuItem>
             </LinkContainer>
           </IfPermitted>
+
+          <MenuItem href={`${this.props.node.transport_address}api-browser`} target="_blank">
```

Comment:
Could you please construct this link like we do with the one in `NodesActions`? Something like:
```javascript
new URI(`${this.props.node.transport_address}/api-browser`).normalizePathname();
```

I think we could add a component that unifies both usages, I leave it up to you.